### PR TITLE
Fixed all links in the documentation

### PR DESCRIPTION
**Pull request by soniasingla**
Signed-off-by: soniasingla <soniasingla.1812@gmail.com>

This PR fixes all the broken links in the Thanos documentation.

CC : @bwplotka @povilasv :hugging_face:

### CODE REVIEW

**Comment by soniasingla** [comment]
Hey @kakkoyun, checking it now. I need to rebase the branch with master (I always forgot this :P)

Also, am not sure why this test `docs / Documentation check` is failing?

**Comment by kakkoyun** [comment]
@soniasingla It says links are broken https://github.com/thanos-io/thanos/pull/2731/checks?check_run_id=747611695

File: CHANGELOG.md [hunk 1]
```diff
@@ -148,7 +148,7 @@ Since there are no consistency guarantees provided by some Object Storage provid
 - [#1870](https://github.com/thanos-io/thanos/pull/1870) UI: Persist settings in query.
 - [#1969](https://github.com/thanos-io/thanos/pull/1969) Sidecar: allow setting http connection pool size via flags.
 - [#1967](https://github.com/thanos-io/thanos/issues/1967) Receive: Allow local TSDB compaction.
-- [#1939](https://github.com/thanos-io/thanos/pull/1939) Ruler: Add TLS and authentication support for query endpoints with the `--query.config` and `--query.config-file` CLI flags. See [documentation](docs/components/rule.md/#configuration) for further information.
```
[the remark this answers is not in the document]
Comment (squat) [reply CHANGELOG.md:151]: Indeed the documentation check in github actions is failing due to this

[the remark this answers is not in the document]
Comment (soniasingla) [reply CHANGELOG.md:165]: @squat I have added the full URLs otherwise the perl script was breaking :)

**Comment by soniasingla** [comment]
@kakkoyun may be we need to configure the perl script, since the changes I made before (without full URLs) were working locally, on GitHub and showing no problems :)

File: CHANGELOG.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (soniasingla) [reply CHANGELOG.md:165]: @squat it's ready for another review 👀

**Comment by povilasv** [comment]
This won't work with browsing files in github markdown as all the links will redirect to webpage.

Let's make it that users can view https://github.com/thanos-io/thanos/blob/master/docs/getting-started.md click links there and don't be redirect to webpage. Like it works right now.

**Pull request closed without merging**